pkgs/servers/openvscode-server/default.nix

```diff
@@ -0,0 +1,187 @@
+    find -name vscode-ripgrep -type d \
+      -execdir mkdir -p {}/bin \; \
+      -execdir ln -s ${ripgrep}/bin/rg {}/bin/rg \;
+
+    # patch shebangs of everything patch shebangs of everything to allow binary packages to build
+    patchShebangs .
+
+    # Playwright is only needed for tests, we can disable it for builds.
+    # There's an environment variable to disable downloads, but the package makes a breaking call to
+    # sw_vers before that variable is checked.
+    patch -p1 -i ${./playwright.patch}
+
+    # rebuild binaries, we use npm here, as yarn does not provide an alternative
+    # that would not attempt to try to reinstall everything and break our
+    # patching attempts
+    npm rebuild --update-binary
+
+    # run yarn install everywhere, skipping postinstall so we can patch in esbuild
+    find . -path "*node_modules" -prune -o \
+      -path "./*/*" -name "yarn.lock" -printf "%h\n" | \
+        xargs -I {} yarn --cwd {} \
+          --frozen-lockfile --offline --ignore-scripts --ignore-engines
+
+    # patch build esbuild
+    mkdir -p build/node_modules/esbuild/bin
+    jq "del(.scripts.postinstall)" build/node_modules/esbuild/package.json | sponge build/node_modules/esbuild/package.json
+    sed -i 's/0.12.6/${esbuild.version}/g' build/node_modules/esbuild/lib/main.js
+    ln -s -f ${esbuild}/bin/esbuild build/node_modules/esbuild/bin/esbuild
+
+    # patch extensions esbuild
+    mkdir -p extensions/node_modules/esbuild/bin
+    jq "del(.scripts.postinstall)" extensions/node_modules/esbuild/package.json | sponge extensions/node_modules/esbuild/package.json
+    sed -i 's/0.11.12/${esbuild.version}/g' extensions/node_modules/esbuild/lib/main.js
+    ln -s -f ${esbuild}/bin/esbuild extensions/node_modules/esbuild/bin/esbuild
+
+    # run postinstall scripts after patching
+    find . -path "*node_modules" -prune -o \
+      -path "./*/*" -name "yarn.lock" -printf "%h\n" | \
+        xargs -I {} sh -c 'jq -e ".scripts.postinstall" {}/package.json >/dev/null && yarn --cwd {} postinstall --frozen-lockfile --offline || true'
+
+    # build vscode-webserver
+    yarn --offline gulp server-min
+  '';
+
+  installPhase = ''
+    mkdir -p $out/libexec/openvscode-server
+
+    cp -R -T ../server-pkg "$out/libexec/openvscode-server"
+
+    makeWrapper "${nodejs}/bin/node" "$out/bin/openvscode-server" \
+      --add-flags "$out/libexec/openvscode-server/out/server.js"
+  '';
+
+  passthru = {
+    prefetchYarnCache = lib.overrideDerivation yarnCache (d: {
+      outputHash = lib.fakeSha256;
+    });
+  };
+
+  meta = with lib; {
+    description = "Run VS Code on a remote server";
+    longDescription = ''
+      vscode-webserver is VS Code running on a remote server, accessible through the
+      browser.
+    '';
+    homepage = "https://github.com/gitpod-io/openvscode-server";
+    license = licenses.mit;
+    maintainers = with maintainers; [ offline ghuntley ];
```

Comment:
✅ Happy to help with maintenance, thanks for fixing the solution to esbuild post install scripts.

Reply:
👍 @offlinehacker I removed you as maintainer from this new package so as not to force the effort of maintenance on you -- happy to add you back if you'd like!